projects/plugins/jetpack/extensions/blocks/subscriptions/edit.js

```diff
@@ -106,9 +106,9 @@ export function SubscriptionEdit( props ) {
 	const activeStyleName = getActiveStyleName( metadata.styles, className );
 
 	const { subscriberCount, subscriberCountString } = useSelect( select => {
-		const { emailSubscribers, socialFollowers } =
+		const { totalSubscribers, socialFollowers } =
 			select( membershipProductsStore ).getSubscriberCounts();
-		let count = emailSubscribers;
+		let count = totalSubscribers;
```

Comment:
This is used for 

![image](https://github.com/user-attachments/assets/9fc8943b-22d5-43f5-acbe-380e4ffcd195)